src/features/iam/iamHelpers.ts

```diff
@@ -22,6 +23,16 @@ async function askForProjectId(): Promise<string | undefined> {
   return project?.projectId
 }
 
+async function enterWallet(): Promise<string | undefined> {
+  const wallet = await window.showInputBox({
+    value: 'http://localhost:3000/holder/claim',
```

Comment:
@carlos-affinidi so... what about Affinidi Wallet? Do we need to obscure it now?

Reply:
@vitalyrudenko-affinidi I believe Carlos is OOO today, should we raise this on the Slack thread instead and merge?